front/app/modules/commercial/machine_translations/api/__mocks__/machineTranslation.ts

```diff
@@ -0,0 +1,9 @@
+export const machineTranslation = {
```

Comment:
Mocks are supposed to match the name of the hook they are mocking and to export a function to that effect. See other mock examples for how it should work.